src/com/project_phone_book/controller/TeleBookController.java

```diff
@@ -0,0 +1,116 @@
+package com.project_phone_book.controller;
+
+import com.project_phone_book.model.Contact;
+import com.project_phone_book.model.TeleBook;
+
+import java.io.File;
+import java.util.Scanner;
+
+public class TeleBookController {
+
+//    public static final int ADD_CONTACT = 0;
+//    public static final int FIND_BY_CHARS = 1;
+//    public static final int DELETE = 2;
+//    public static final int END = 3;
+
+    TeleBook teleBook = new TeleBook();
+    Scanner sc = new Scanner(System.in);
+    FileOperations file = new FileOperations();
+
+
+    public void phoneBookApi() {
+        createFileAndAddToList();
+        Option option = null;
+        while (option != Option.END) {
+            intro();
```

Comment:
intro is not a good name for method. It should be more descriptive - printIntro for example